File: RSPAClient/UserInterface/AtPar.Web/src/app/Entities/MT_ATPAR_MESSAGE_TBL.ts
export class MT_ATPAR_MESSAGE_TBL {
    public MESSAGE_ID: number;
    public FROM_USER: string;
    public TO_USER: string;
    public MESSAGE_TEXT: string;
    public CREATED_DATE?: Date;
    public SERVER_DELIVERED_DATE?: Date;
    public FINAL_DELIVERED_DATE?: Date;
    public READ_DATE?: Date;
    public DELETED_DATE?: Date;
    public STATUS: string;
    public LAST_UPDATE_DATE?: Date;
}

File: RSPAClient/Core/AtPar.POCOEntities/MT_ATPAR_MESSAGE_TBL.cs
using System;
using System.Collections.Generic;

namespace AtPar.POCOEntities
{
    public partial class MT_ATPAR_MESSAGE_TBL
    {
        public int MESSAGE_ID { get; set; }
        public string FROM_USER { get; set; }
        public string TO_USER { get; set; }
        public string MESSAGE_TEXT { get; set; }
        public Nullable<System.DateTime> CREATED_DATE { get; set; }
        public Nullable<System.DateTime> SERVER_DELIVERED_DATE { get; set; }
        public Nullable<System.DateTime> FINAL_DELIVERED_DATE { get; set; }
        public Nullable<System.DateTime> READ_DATE { get; set; }
        public Nullable<System.DateTime> DELETED_DATE { get; set; }
        public string STATUS { get; set; }
        public Nullable<System.DateTime> LAST_UPDATE_DATE { get; set; }
    }
}
